serenity-automation-mentoring/serenity.properties

```diff
@@ -0,0 +1,20 @@
+# Define the default driver
+webdriver.driver=chrome
+webdriver.chrome.driver=C:\\Users\\Simba\\Downloads\\chromedriver_last\\chromedriver.exe
```

Comment:
Do not use absolute paths in configs under git. Use only relative paths, which will 100% work at every machine

Reply:
Done. Was done as well as in kf-rest-assured-atg-api-tests